backend/test/api/base_test.clj

```diff
@@ -9,15 +9,15 @@
 (ctest/use-fixtures :each mocking/mock-persistence)
 
 (defexpect game-creation
-  (expect 0 (:game-id (base/create-game)))
+  (expect number? (:game-id (base/create-game)))
```

Comment:
It would probably be interesting to see that creating *another* game, gives a different :game-id

Reply:
Agree!